packages/themes/scss/_mixins.scss

```diff
@@ -0,0 +1,74 @@
+// Code generated by @carbon/themes. DO NOT EDIT.
+//
+// Copyright IBM Corp. 2018, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+
+@import './theme';
+@import './variables';
```

Comment:
I don't think `@import './variables';` is necessary as _mixins.scss does not reference anything from _variables.scss.